HandleNHCreateOrUpdateInstallationCallActivity/__tests__/handler.test.ts

```diff
@@ -74,7 +72,7 @@ describe("HandleNHCreateOrUpdateInstallationCallActivity", () => {
     expect.assertions(3);
 
     const res = await handler(contextMock as any, input);
-    expect(res.kind).toEqual("SUCCESS");
+    expect(ActivityResultSuccess.is(res)).toBeTruthy();
```

Comment:
This is more personal taste than a strong opinion:

`expect(foo).toBe(true)` is more readable than `expect(foo).toBeTruthy()`. Also, note that they have different meaning (example: `1` is _truthy_ but it's not `true`).